src/plugins/data_source_management/public/components/data_source_selectable/data_source_selectable.tsx

```diff
@@ -180,6 +183,9 @@ export class DataSourceSelectable extends React.Component<
         'title',
         'auth.type',
       ]);
+      if (fetchedDataSources?.length === 0 && this.props.hideLocalCluster) {
```

Comment:
instead of evaluate fetechedDataSources, we should evaluate the options after filter, right?

Reply:
yeah, updated